Dockerfile.windows

```diff
@@ -0,0 +1,20 @@
+FROM microsoft/dotnet:1.0.0-preview2-windowsservercore-sdk
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+# REMARK: Temporary workaround for Windows DNS client weirdness
+RUN set-itemproperty -path 'HKLM:\SYSTEM\CurrentControlSet\Services\Dnscache\Parameters' -Name ServerPriorityTimeLimit -Value 0 -Type DWord
+
+RUN New-Item -Path \MusicStore\samples\MusicStore.Standalone -Type Directory
+WORKDIR MusicStore
+
+ADD samples/MusicStore.Standalone/project.json samples/MusicStore.Standalone/project.json
+ADD NuGet.config .
+RUN dotnet restore --no-cache .\samples\MusicStore.Standalone
```

Comment:
`--no-cache` actually means don't use the cached http response from last restore and hit the http endpoint directly to ensure the latest packages are pulled. It doesn't mean packages are not stashed in `~/.nuget` folder. You can do a little experiment here. Delete the `~/.nuget` folder and run `dotnet restore --no-cache`. The packages are downloaded regardless.

In order to save the disk, you need to remove the `~/.nuget` as well as the `%LocalAppData%/NuGet/Cache` folder.

I agree the description is misleading here.